pubspec.yaml

```diff
@@ -10,7 +10,7 @@ dependencies:
     git: https://github.com/dart-atom/atom.dart.git
   path: ^1.3.0
   pub_semver: ^1.2.1
-  serial_port: ^0.3.1
+  #serial_port: ^0.3.1 << bundled
```

Comment:
Hmmm. I naively just bundled the package and manually created/updated a `.packages` file. It needs to be there for the command line utility, but this approach won't work. I think I will just merge the package into the code base for now to get things working. Long term I want to find a better solution.